### ISSUE TITLE
Style settings "Slurs & ties": Values in the spin boxes cannot be changed using the cursor keys

### ISSUE THREAD

**Issue by Christian150373** [musescore/MuseScore#30208]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

Go to Style -> Slurs & ties
Try to change the values in the spin boxes with the cursor keys
Values can be changed using the up and down arrows

### Supporting files, videos and screenshots

<img width="549" height="502" alt="Image" src="https://github.com/user-attachments/assets/d78945d9-d229-4ee5-8806-e8d424a0fcc1" />

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 24H2 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.6.0-252730940, revision: github-musescore-musescore-7829779

### Regression

I was unable to check

### Operating system

Windows 11

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by ghost** [comment]
A 4.6 regression; @cbjeukendrup perhaps via https://github.com/musescore/MuseScore/pull/29981?

**Comment by cbjeukendrup** [comment]
I'm afraid this is because it's a QML page embedded in a QWidget dialog, and we have no solution for keyboard navigation in that situation.

**Comment by Eism** [comment]
Yes, changing using the cursor keys works through the navigation system, which works through the shortcut system, which [is disabled](https://github.com/musescore/MuseScore/blob/master/src/framework/shortcuts/internal/shortcutscontroller.cpp#L32) for widget dialogs.

**Comment by bkunda** [comment]
To be solved when the Style dialog gets overhauled.
